### PR TITLE
chore(docs): reorganize contribution guide

### PR DESCRIPTION
**Pull request by eladb**
Reorganize contribution guide and merge RFCs into it (RFCs are not meant to be references for users, just for contributors and maintainers).

*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

**Review by ShaiBer — APPROVED** [review]

File: docs/03-concepts/02-inflight.md [hunk 1]
```diff
@@ -49,3 +49,106 @@ Wing is designed around this new programming paradigm, which we call
...
+Processes](https://www.erlang.org/doc/reference_manual/processes.html) and the recently popularized,
+cooperative concurrency with `async/await`, developers have been looking for ways to write
+applications that included more than a single thread of execution in a safe and intuitive manner.
+
+Wing's inflight functions can be thought of as a type of asynchronous function that can be
+**packaged and executed on a remote system**, such as inside a [Kubernetes
+Pod](https://kubernetes.io/docs/concepts/workloads/pods/), [AWS
+Lambda](https://aws.amazon.com/lambda/), [Azure
+Functions](https://azure.microsoft.com/en-us/products/functions/), on [Cloudflare Edge
+Workers](https://workers.cloudflare.com/) or on any other cloud compute platform.
```
Comment (Chriscbr) [inline docs/03-concepts/02-inflight.md:76]: I wonder if you want to add a sentence like

> While inflights are typically compiled into machine code (like JavaScript), Wing may also be able to compile them into state machines and other formats in the future.

**Review by Chriscbr — APPROVED** [review]
LGTM

File: docs/03-concepts/02-inflight.md (hunk 1 above)
Comment (ShaiBer) [reply docs/03-concepts/02-inflight.md:76]: I think we should

**Review by staycoolcall911 — COMMENTED** [review]
Great! @eladb, while you're at it - could you please also move the wingsdk spec to be next to the language spec? It is currently under the RFCs folder

**Pull request merged**